README.md

```diff
@@ -44,22 +45,33 @@ used to provision this module:
 
 ## Usage
 
-The **terraform-enterprise** module can be used through the quickstart
-method or by including it in a Terraform configuration.
+The **terraform-enterprise** module can be used through either the
+[Quickstart Method](#quickstart-method) or the
+[Terraform Configuration Method](#terraform-configuration-method).
 
-### Quickstart
+### Quickstart Method
 
-The quickstart method provides a ready-to-apply Terraform configuration
-which includes the **terraform-enterprise** module. More information is
-available in the [quickstart repository][quickstart].
+This Git repository can be used directly to deploy Terraform Enterprise
+Clustered without writing any Terraform configuration. All that is
+required is to clone this repository, change the working directory to
```

Comment:
Since this is going to be in the registry by virtue of README, do we want to link to the repo  with a self link at the bottom so there's an easy click-thru to the repo that needs to be cloned?

```suggestion
required is to clone [this repository](self), change the working directory to
```

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/66/commits/c4980e7ea8b6b356698297028d86fd0380108350.